jupyter-extension/jupyterlab_pachyderm/handlers.py

```diff
@@ -229,7 +220,43 @@ async def get(self, path):
         content = int(content)
 
         model = await ensure_async(
```

Comment:
It looks like we've axed a lot of the asynchronous calls within these handlers. Do we still need this `ensure_async`?